php/pantheon/checks/objectcache.php

```diff
@@ -29,11 +29,13 @@ public function run() {
 		} else {
 			$this->alerts[] = array("message"=> "object-cache.php exists", "code" => 0);
 		}
-
-		if ( ! defined( 'WP_REDIS_OBJECT_CACHE' ) || ! WP_REDIS_OBJECT_CACHE ) {
-			$this->alerts[] = array("message"=> 'Use Redis with the WP Redis object cache drop-in to speed up your backend. <a href="https://pantheon.io/docs/wordpress-redis/" target="_blank">Learn More</a>', "code" => 1);
+		if ( defined( 'WP_REDIS_OBJECT_CACHE' )) {
+			$this->alerts[] = array('message'=> 'WP Redis for object caching was found. We recommend using Object Cache Pro as a replacement. <a href="https://docs.pantheon.io/guides/object-cache-pro/installing-configuring/" target="_blank">Learn More</a> about how to install Object Cache Pro.', 'code' => 1);
+		}
+		elseif ( ! defined( 'WP_REDIS_CONFIG' ) || ! WP_REDIS_CONFIG ) {
```

Comment:
Nit: This could be bumped up to the previous line. It's not syntactically wrong, it's just prettier. Not a blocker. ✅